src/app/components/SearchForm/SearchForm.jsx

```diff
@@ -15,6 +16,21 @@ class SearchForm extends React.Component {
     this.submitSearchRequest = this.submitSearchRequest.bind(this);
   }
 
+  /**
+   * Used to update the downstream props updated by the
+   * parent component, SearchContainer.
+   *
+   * @param {object} nextProps
+   */
+  componentWillReceiveProps(nextProps) {
```

Comment:
This component could also benefit from a `window.scrollTo()` call. When I am on a Work Detail page and I click on a subject or author link, the new search works well but the page doesn't go to the top of the results. _Maybe_ it would be better if the first result is focused or maybe a text descriptor explaining the new set of results (e.g. the "Displaying 1 - 2 ....").

Reply:
The set of components that handle messaging, sorting, paging still need work which is why the message is so generic for now. Definitely wasn't sure the best way to handling scrolling either:  what method is best, what components need it, etc.

Reply:
The message being generic is okay. The UX team can handle where to focus to in that case to see what method is best.